.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,8 +30,17 @@
 1. ...
 2. ...
 
+### Release notes
+
+For user-facing changes, add a meaningful release note. For examples, see [Magento Cloud ECE-tools release notes](https://devdocs.magento.com/cloud/release-notes/ece-release-notes.html).
+
+### Associated documentation updates
+<!--
+ If your proposed update requires user documentation, submit a PR to the Magento DevDocs repository. For extensive updates requiring assistance, submit an issue to DevDocs. See https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md.
+ -->
+Add link to Magento DevDocs PR or Issue, if needed.
+
 ### Contribution checklist
  - [ ] Pull request has a meaningful description of its purpose
+ - [ ] Pull request introduces user-facing changes and includes meaningful release notes and documentation
  - [ ] All commits are accompanied by meaningful commit messages
- - [ ] All new or changed code is covered with unit/integration tests (if applicable)
- - [ ] All automated tests passed successfully (all builds on Travis CI are green)
```

Comment:
Why we removed these 2 items?

Reply:
Restored these items. This was a copy/paste error.  The PR templates in the Cloud docker, patches, and components repositories do not have these items in the checklist.  Do they need to be added to those PR templates?

Reply:
It's OK not to have it for docker PRs and MCC